src/getPostcssResult.js

```diff
@@ -43,13 +44,29 @@ export default function (
       } else if (syntax === "sugarss" || !syntax && fileExtension === ".sss") {
         syntax = sugarssSyntax
       } else if (syntax) {
-        throw new Error("You must use a valid syntax option, either: scss, less or sugarss")
+        try {
+          // Should check these possibilities:
+          //   a. name of a node_module
+          //   b. absolute path
+          //   c. relative path relative to `process.cwd()`.
+          // If none of the above work, we'll try a relative path starting
+          // in `process.cwd()`.
+          const syntaxModule = resolveFrom(process.cwd(), syntax)
```

Comment:
This is the same logic used in `cli.js` to resolve the configFile module. Would you prefer this was extracted into a utility method somewhere, or is the duplication okay?